### DISCUSSION TITLE
Import currency field

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3775] [Q&A]
Hi,

I am importing csv file having a currency field with $ value (for e.g $90). 
But the currency field is showing $0.00 in all the table rows instead of the actual value.
Can you please help in resolving this?

Thanks and Regards,
Sumathy Venugopal

**Comment by wingkwong** [comment]
Can you share a sample csv file and your project info?

It looks fine on my side.

![image](https://user-images.githubusercontent.com/35857179/192078650-2a0c3571-40be-4ea6-a351-a8986e038194.png)

```
Title,currency
A,$10.00
B,$20.00
C,10
D,20
E,30.00
```

**Reply by wingkwong** [reply]
@MGW-SumathyV any updates?
